toolchain/lower/testdata/function/generic/call_basic.carbon

```diff
@@ -0,0 +1,122 @@
+// Part of the Carbon Language project, under the Apache License v2.0 with LLVM
+// Exceptions. See /LICENSE for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+// AUTOUPDATE
+// TIP: To test this file alone, run:
+// TIP:   bazel test //toolchain/testing:file_test --test_arg=--file_tests=toolchain/lower/testdata/function/generic/call_basic.carbon
+// TIP: To dump output, run:
+// TIP:   bazel run //toolchain/testing:file_test -- --dump_output --file_tests=toolchain/lower/testdata/function/generic/call_basic.carbon
+
+fn F[T:! type](x: T) {
+}
+
+fn H[T:! type](x: T) {
+}
+
+fn G[T:! type](x: T) -> T {
+  // TODO: call crashing.
```

Comment:
Could you add a few more words here?

I guess the commented out call is crashing in lowering?